src/main/java/ladder/Name.java

```diff
@@ -0,0 +1,41 @@
+package ladder;
+
+import java.util.Objects;
+
+public class Name {
+    public static final int MAX_NAME_LENGTH = 5;
+    private String name;
```

Comment:
```suggestion
public class Name {
    public static final int MAX_NAME_LENGTH = 5;
    private final String name;
```

가능한 내부 멤버에는 final 키워드를 붙여서, 불변 객체를 프로그래밍 언어적으로 보장할 수 있도록 해주시면 감사하겠습니다.

Reply:
넵! 불변객체를 만드는걸 또 깜빡해버렸네요 ㅠ